### DISCUSSION TITLE
Support for matching IDs on explore page

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#995] [Announcements]
Hi all,

I made a tweak at https://github.com/pi-base/web/pull/201 which should allow you to match on ID numbers at https://topology.pi-base.org/spaces - the "filter by text" will match S000123 if you type 123, and "filter by formula" will match P000123 if you type 123.

Let me know if you see any issues.

**Comment by prabau** [comment]
I entered "33" in the "filter by text" box and one of the choices returned is S171 (because the text for it has 33 as part of a mathoverflow id).
Not sure if it's a problem, but just letting you know.

**Reply by StevenClontz** [reply]
Thanks - for now as long as the top result is S33 (which I think it is) I think this is fine.

**Comment by Moniker1998** [comment]
I would like something like this, but for these three tabs:
https://topology.pi-base.org/spaces/all
https://topology.pi-base.org/properties
https://topology.pi-base.org/theorems
Currently this only works for https://topology.pi-base.org/spaces

**Reply by StevenClontz** [reply]
I think I have this implemented at https://github.com/pi-base/web/pull/210

**Reply by StevenClontz** [reply]
Now live, e.g. https://topology.pi-base.org/spaces/all?filter=123
